libraries/default.rb

```diff
@@ -0,0 +1,15 @@
+def credential_secrets(databag = node['osl-git']['secrets_databag'], item = node['osl-git']['secrets_item'])
```

Comment:
You may want to rename this to something like ``git_credential_secrets`` in case it overrides what we have in osl-openstack. I just realize that might be a problem but I don't know how scope works in this case.